sdk/tests/js/webgl-test-utils.js

```diff
@@ -1767,6 +1767,45 @@ var shouldGenerateGLErrorIn = function(gl, expectedErrorList, evalStr) {
 };
 
 /**
+ * Tests that a function throws or not.
+ * @param {!WebGLContext} gl The WebGLContext to use.
+ * @param throwType Type of thrown error (e.g. TypeError), or false.
+ * @param {string} info Info on what's being tested
+ * @param {function} func The func to test.
+ */
+var shouldThrow = function(gl, throwType, info, func) {
```

Comment:
It's confusing that "shouldThrow" handles assertions both that the callee throws and doesn't. What about adding two functions "shouldThrow" and "shouldNotThrow" with different signatures and making this one an implementation function instead?